packages/context/test/acceptance/extension-point.ts

```diff
@@ -0,0 +1,215 @@
+// Copyright IBM Corp. 2013,2017. All Rights Reserved.
+// Node module: loopback
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {expect} from '@loopback/testlab';
+import {Context, Binding, BindingScope, inject} from '../..';
+
+// tslint:disable:no-any
+/**
+ * Interface for the extension point configuration
+ */
+export interface ExtensionPointConfig {
+  extensions: {
+    [extensionName: string]: any;
+  };
+  extensionPoint: {
```

Comment:
What is the point of this member? AFAICT, it's not used anywhere else in this pull request.

Reply:
I was trying to separate config properties for the extension point itself from the ones for extensions.